_developers/score-core.md

```diff
@@ -69,3 +69,11 @@ Adjusts the height of the score screen in pixels.
 ```javascript
 Materia.ScoreCore.setHeight(350)
 ```
+
+## Materia.ScoreCore.getMediaUrl
+
+Convert a materia asset id into a url.
```

Comment:
Capitalize 'Materia'?

Reply:
Good idea, I did a global sweep for lower case materias